gradio/blocks.py

```diff
@@ -912,6 +912,7 @@ def queue(
             update_intervals=status_update_rate if status_update_rate != "auto" else 1,
             max_size=max_size,
         )
+        self.config = self.get_config_file()
```

Comment:
So that `queue_enabled` shows up in the config if you call `queue` outside of a context manager and without calling `launch`.

Reply:
To confirm, is this just needed for the new test in `test/test_blocks`?

Reply:
It's so that the queue is enabled in the config when you do `blocks.queue()` but don't call `.launch()`